src/commands/run/spec/runCommand.spec.ts

```diff
@@ -0,0 +1,115 @@
+import * as runModule from '../run'
+import { RunCommand } from '../runCommand'
+import { Logger, LogLevel, ServerType, Target } from '@sasjs/utils'
+import * as configUtils from '../../../utils/config'
+import { ReturnCode } from '../../../types/command'
+import { setConstants } from '../../../utils'
+
+const defaultArgs = ['node', 'sasjs']
+const target = new Target({
+  name: 'test',
+  appLoc: '/Public/test/',
+  serverType: ServerType.SasViya,
+  contextName: 'test context'
+})
+const sasFilePath = 'path/to/soure.sas'
+describe('RunCommand', () => {
+  beforeAll(async () => {
+    await setConstants()
+  })
+
+  beforeEach(() => {
+    setupMocks()
+  })
+
+  it('should parse sasjs run command', async () => {
+    const args = [...defaultArgs, 'run', sasFilePath]
+
+    const command = new RunCommand(args)
```

Comment:
would be nice to put it in function to reduce code duplication

Reply:
refactored specs, fixed.